modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/StatisticsUtils.java

```diff
@@ -172,4 +174,61 @@ public static StatisticsTarget statisticsTarget(StatisticsKeyMessage msg) {
         String[] cols = (msg.colNames() == null) ? null : msg.colNames().toArray(new String[0]);
         return new StatisticsTarget(msg.schema(), msg.obj(), cols);
     }
+
+    /**
+     * Test if specified statistics is fit to all required versions.
+     * It means that specified statistics contains all columns with at least versions
+     * from specified map.
+     *
+     * @param stat Statistics to check. Can be {@code null}.
+     * @param versions Map of column name to required version.
+     * @return {@code true} if it is, {@code false} otherwise.
```

Comment:
actually the method returns integer value

Reply:
Fixed.